downstream/modules/platform/ref-single-controller-hub-eda-with-managed-db.adoc

```diff
@@ -101,4 +104,61 @@ sso_keystore_password=''
 # automationedacontroller_ssl_cert=/path/to/automationeda.crt
 # automationedacontroller_ssl_key=/path/to/automationeda.key
 
+## Operational parameters
+automationedacontroller_postgresql_version: '13'
+automationedacontroller_redis_database: '4'
+
+automationedacontroller_allowed_addresses:
+  - "{{ ansible_all_ipv4_addresses | default([]) }}"
+  - "{{ ansible_all_ipv6_addresses | default([]) }}"
+
+_automationedacontroller_trusted_origins: "{{ [automationedacontroller_disable_https | ternary('http://', 'https://')] | product(_automationedacontroller_allowed_hostnames | flatten) | map('join') | list | join(',') }}"
+
+automationedacontroller_container_registry_credential: "Decision Environment Container Registry"
+decision_environments:
+  - name: "Default Decision Environment"
+    image: "{{ _ee_image_base }}de-supported-rhel8:latest"
+    credential_name: "{{ automationedacontroller_container_registry_credential }}"
+
+automationedacontroller_db_settings:
+  DB_NAME: '{{ automationedacontroller_install_pg_database }}'
+  DB_HOST: '{{ automationedacontroller_install_pg_host }}'
+  DB_PORT: '{{ automationedacontroller_install_pg_port | int }}'
+  DB_USER: '{{ automationedacontroller_install_pg_username }}'
+  DB_PASSWORD: '{{ automationedacontroller_install_pg_password }}'
+  DB_SSL_MODE: '{{ _automationedacontroller_pg_sslmode }}'
+
+automationedacontroller_gw_settings:
+  ANSIBLE_BASE_JWT_VALIDATE_CERT: true
+  ANSIBLE_BASE_JWT_REDIRECT_TYPE: 'eda'
+  REST_FRAMEWORK:
+    DEFAULT_AUTHENTICATION_CLASSES:
+      - 'rest_framework.authentication.SessionAuthentication'
+      - 'ansible_base.jwt_consumer.eda.auth.EDAJWTAuthentication'
+      - 'rest_framework.authentication.BasicAuthentication'
+
+## Configurable parameters
```

Comment:
@Alex-Izquierdo - I added the code from https://github.com/ansible/automation-platform-collection/blob/devel/roles/automationedacontroller/defaults/main.yml#L38 to the existing code sample in this section. Is that correct?